tools/accuracy_checker/configs/face-detection-0105.yml

```diff
@@ -0,0 +1,39 @@
+models:
+  - name: face-detection-0105
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/face-detection-0105/FP32/face-detection-0105.xml
+        weights: intel/face-detection-0105/FP32/face-detection-0105.bin
+        adapter:
+          type: fcos_person
```

Comment:
hmm... looks like we need to rename adapter to `fcos_single_class` since it used not only for person detection

Reply:
actually this even more common than fcos only. renamed it to 'class_agnostic_detection'